iocore/net/UnixNetAccept.cc

```diff
@@ -221,11 +247,26 @@ NetAccept::init_accept_per_thread()
   }
 
   period = -HRTIME_MSECONDS(net_accept_period);
-  n      = eventProcessor.thread_group[opt.etype]._count;
+  REC_ReadConfigInteger(accept_threads, "proxy.config.accept_threads");
+  if (accept_threads == 0) {
+    n = eventProcessor.thread_group[ET_ACCEPT]._count;
+  } else {
+    ink_assert(accept_threads < 0);
+    n = -accept_threads;
+  }
 
   for (i = 0; i < n; i++) {
-    NetAccept *a       = (i < n - 1) ? clone() : this;
-    EThread *t         = eventProcessor.thread_group[opt.etype]._thread[i];
+    NetAccept *a = (i < n - 1) ? clone() : this;
+#ifdef SO_REUSEPORT
+    if (a != this) {
+      a->server.fd = NO_FD;
+      if (a->do_listen(NON_BLOCKING)) {
+        delete a;
```

Comment:
Should we log a error message or destroyed a NetAccept when listen failed.
It seems an empty thread will be created.